src/components/slider-panel/index.js

```diff
@@ -87,6 +92,12 @@ class SliderPanel extends Component {
 						onChange={ () => setAttributes( { pageDots: ! pageDots } ) }
 						help={ this.getDotNavigationHelp }
 					/>
+					<ToggleControl
+						label={ __( 'Align Cells' ) }
+						checked={ !! alignCells }
+						onChange={ () => setAttributes( { alignCells: ! alignCells } ) }
```

Comment:
If you pass the `this.flkty` as a prop down from `edit.js` down through the `Inspector` component and then into the `SliderPanel` component, you can use it in this onChange to set the `cellAlign` option of Flickity. This resolves the issue with it not updating in the editor but there are some display issues.

We need to find a way to re-render the Flickity component entirely but there is no obvious method to call. I also expect React would do this behind the scenes since properties are changing.